### PR TITLE
S5t3 display lives

### PR DESCRIPTION
**Pull request by lilyboughton**
Adds the lives counter to the game screen.
Should not appear on the splash screen.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -173,6 +174,26 @@ div[data-hit ~= "2"]{
...
+}
+
+.livesDisplay {
+
+}
+
+.lynxFoundDisplay {
+
+}
+
```
Comment (Reductito) [inline styles.css]: not needed can take out

File: index.js [hunk 2]
```diff
@@ -96,6 +96,16 @@ function turnCard(numLynx, numSnakes, lives) {
                 const modalText = 'Too many snake bites... Sorry!'
                 decideOutcome(modalText)
             }
+
+            //updates lives bar
+            if(lives === 1){
```
Comment (Reductito) [inline index.js]: space after if and before curly brace

File: index.js [hunk 3]
```diff
@@ -135,7 +148,21 @@ document.querySelector('.form').addEventListener('submit', (e) => {
...
     const gridSize = parseInt(document.getElementById('gridSize').value)
     const numSnakes = gridSize - numLynx
+
+    //displays lives counter bar
+    document.getElementById('livesCounterBar').style.display = 'flex'
+
     let lives = generateLives(numLynx)
+
+    //updates lives bar
+    if(lives === 1){
```
Comment (Reductito) [inline index.js]: space after if and before curly brace

**Review by Reductito — COMMENTED** [review]
Only saw tiny things, just some spaces needed here and there, can also look to remove unneeded empty lines in html and js

**Comment by TimTapley** [comment]
Media breaks not set on attempts display bar. Probably needs to deploy at around 650?

**Comment by Reductito** [comment]
Looks good!

**Pull request merged**